lib/media/track/mediatrack.js

```diff
@@ -145,27 +151,35 @@ class MediaTrack extends Track {
   _end() {
     this._log.debug('Ended');
     if (this._dummyEl) {
-      this._detachElement(this._dummyEl);
+      this._detachElementInternal(this._dummyEl);
+      this._dummyEl.remove();
+      this._dummyEl.srcObject = null;
       this._dummyEl.oncanplay = null;
+      this._dummyEl = null;
     }
```

Comment:
Nice one @makarandp0 . So we are only detaching elements, not properly disposing them. I've learned that we also need to pause the element first, then, call load after setting the srcObject to null. Something like

```js
this._detachElementInternal(this._dummyEl);
this._dummyEl.pause();
this._dummyEl.remove();
this._dummyEl.srcObject = null;
this._dummyEl.oncanplay = null;
this._dummyEl.load();
this._dummyEl = null;
```

Reply:
okay, I will update and test per your suggestion.

Reply:
After experimenting a bit, I found that just setting srcObject to null is sufficient to avoid this issue. I will keep this simple and just reset `srcObject` as [suggested](https://bugs.chromium.org/p/chromium/issues/detail?id=1232649#c8) as a workaround: